upgrade_tests/test_scenarios.py

```diff
@@ -0,0 +1,251 @@
+# -*- encoding: utf-8 -*-
+"""Test for Upgrade Scenario's
+
+:Requirement: Satellite upgrade
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import os
+import time
+
+from automation_tools.satellite6 import hammer
+from upgrade.helpers.docker import (
+    docker_execute_command,
+    refresh_subscriptions_on_docker_clients
+)
+from upgrade_tests.helpers.scenarios_utils import (
+    create_dict,
+    dockerize,
+    get_entity_data,
+    get_latest_repo_version,
+    get_satellite_host,
+    rpm1,
+    rpm2
+)
+from fabric.api import execute, run
+from fauxfactory import gen_alpha
+from unittest2.case import TestCase
+from upgrade_tests import post_upgrade, pre_upgrade
+
+
+class ScenarioBug1429201(TestCase):
+    """This Class will server as a whole scenario with pre-upgrade and
+    post-upgrade test-case
+    """
+    prd_name = gen_alpha()
+    repo_name = gen_alpha()
+    lc_name = gen_alpha()
+    ak_name = gen_alpha()
+    cv_name = gen_alpha()
+    docker_vm = os.environ.get('DOCKER_VM')
+    org_id = 1
+    sat_host = get_satellite_host()
+    file_path = '/var/www/html/pub/custom_repo/'
+    custom_repo = 'https://' + sat_host + '/pub/custom_repo/'
+    _, rpm1_name = os.path.split(rpm1)
+    _, rpm2_name = os.path.split(rpm2)
+
+    def setUp(self):
+        hammer.set_hammer_config()
+
+    @pre_upgrade
+    def test_pre_user_scenario_bug_1429201(self):
+        """This is pre-upgrade scenario test to verify if we can create a
+         custom repository and consume it via client
+
+         :id: 8fb8ec87-efa5-43ed-8cb3-960ef9cd6df2
+
+         :steps:
+             1. Create repository RepoFoo that you will later add to your
+                Satellite. This repository should contain PackageFoo-1.0.rpm
+             2. Install satellite 6.1
+             3. Create custom product ProductFoo pointing to repository RepoFoo
+             4. Sync RepoFoo
+             5. Create content view CVFoo
+             6. Add RepoFoo to CVFoo
+             7. Publish version 1 of CVFoo
+
+         :expectedresults: The client and product is created successfully
+         """
+        try:
+            run('rm -rf {0}'.format(self.file_path))
+            run('mkdir {0}'.format(self.file_path))
+        except OSError:
+            run('mkdir /var/www/html/pub/custom_repo')
+        run('wget {0} -P {1}'.format(rpm1, self.file_path))
+        run('createrepo --database {0}'.format(self.file_path))
+        # End to End product + ak association
+        print hammer.hammer_product_create(self.prd_name, self.org_id)
```

Comment:
This way the stdouts on job run will increase, I am really against of putting so ma ny print statements. 
The stdouts will be huge with increasing test scenarios. Please remove them.